doc/man1/flux-kvs.adoc

```diff
@@ -50,13 +50,15 @@ interpreted as encoded JSON and formatted accordingly.  If '-r', value
 is displayed without a newline.  If '-t', the RFC 11 object is displayed.
 '-a treeobj' causes the lookup to be relative to an RFC 11 snapshot reference.
 
-*put* [-j|-r|-t] [-n] 'key=value' ['key=value...']::
+*put* [-j|-r|-t] [-n] [-A] 'key=value' ['key=value...']::
 Store 'value' under 'key' and commit it.  If it already has a value,
 overwrite it.  If no options, value is stored directly.  If '-j', it is
 first encoded as JSON, then stored.  If '-r', the value may be read from
 standard input if specified as "-", and may include embedded NULL bytes.
 If '-t', value is stored as a RFC 11 object.  '-n' prevents the commit
-from being merged with with other contemporaneous commits.
+from being merged with with other contemporaneous commits.  '-A' appends the
+value to a key instead of overwriting the value.  Append only works with the
```

Comment:
should work with no options, if my suggestion is accepted.   JSON is the one that gets weird since a JSON object appended to a JSON object isn't valid JSON.

Reply:
Ahh, I didn't adjust given changes with #1262.  Will have to go back and change that.